indico/migrations/versions/20230424_1511_cd7038a37646_add_notes_column_to_room_bookings.py

```diff
@@ -0,0 +1,24 @@
+"""Add notes column to room bookings
+
+Revision ID: cd7038a37646
+Revises: 5d05eda06776
+Create Date: 2023-04-24 15:11:55.407209
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = 'cd7038a37646'
+down_revision = '5d05eda06776'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('reservations', sa.Column('notes', sa.Text(), nullable=True), schema='roombooking')
```

Comment:
We usually store an empty string (default) and make columns not nullable. In the revision you need to create it with `serer_default=''` and then drop the server_default again right below (there are a bunch of examples if you search for `server_default=None`)